{"uid":"8f979bf0657593ab","name":"test_remember_password","fullName":"case.huaxin.test_aasmoke.TestSmoke#test_remember_password","historyId":"87d9b0e1ae25697a10a133d7f73c8f23","time":{"start":1665455165168,"stop":1665455167286,"duration":2118},"description":"  测试步骤七：是否记住密码，判断退出登录后，密码框是否存在密码 ","descriptionHtml":"<p>测试步骤七：是否记住密码，判断退出登录后，密码框是否存在密码</p>\n","status":"passed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":3,"retriesStatusChange":true,"beforeStages":[{"name":"_xunit_setup_class_fixture_TestSmoke","time":{"start":1665455053351,"stop":1665455090676,"duration":37325},"status":"passed","steps":[],"attachments":[],"parameters":[],"attachmentsCount":0,"shouldDisplayMessage":false,"stepsCount":0,"hasContent":false}],"testStage":{"description":"  测试步骤七：是否记住密码，判断退出登录后，密码框是否存在密码 ","status":"passed","steps":[],"attachments":[{"uid":"87f816f813140a10","name":"log","source":"87f816f813140a10.txt","type":"text/plain","size":119},{"uid":"f2a2f78359b0302c","name":"stderr","source":"f2a2f78359b0302c.txt","type":"text/plain","size":162}],"parameters":[],"attachmentsCount":2,"shouldDisplayMessage":false,"stepsCount":0,"hasContent":true},"afterStages":[{"name":"_xunit_setup_class_fixture_TestSmoke::0","time":{"start":1665455167287,"stop":1665455169747,"duration":2460},"status":"passed","steps":[],"attachments":[],"parameters":[],"attachmentsCount":0,"shouldDisplayMessage":false,"stepsCount":0,"hasContent":false}],"labels":[{"name":"severity","value":"trivial"},{"name":"feature","value":"冒烟测试"},{"name":"epic","value":"约克智慧家"},{"name":"story","value":"记住密码"},{"name":"parentSuite","value":"case.huaxin"},{"name":"suite","value":"test_aasmoke"},{"name":"subSuite","value":"TestSmoke"},{"name":"host","value":"LAPTOP-TABJSFGD"},{"name":"thread","value":"11600-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"case.huaxin.test_aasmoke"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"trivial","retries":[{"uid":"d78fe882e1c7b8c7","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to remote server. Original error: Error: socket hang up\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to remote server. Original error: Error: socket hang up\n    at JWProxy.command$ (C:\\Program Files (x86)\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\jsonwp-proxy\\proxy.js:238:13)\n    at tryCatch (C:\\Program Files (x86)\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\babel-runtime\\regenerator\\runtime.js:67:40)\n    at GeneratorFunctionPrototype.invoke [as _invoke] (C:\\Program Files (x86)\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\babel-runtime\\regenerator\\runtime.js:315:22)\n    at GeneratorFunctionPrototype.prototype.(anonymous function) [as throw] (C:\\Program Files (x86)\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\babel-runtime\\regenerator\\runtime.js:100:21)\n    at GeneratorFunctionPrototype.invoke (C:\\Program Files (x86)\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\babel-runtime\\regenerator\\runtime.js:136:37)\n    at run (C:\\Program Files (x86)\\Appium\\resources\\app\\webpack:\\~\\babel-polyfill\\~\\core-js\\modules\\es6.promise.js:75:1)\n    at C:\\Program Files (x86)\\Appium\\resources\\app\\webpack:\\~\\babel-polyfill\\~\\core-js\\modules\\es6.promise.js:92:1\n    at flush (C:\\Program Files (x86)\\Appium\\resources\\app\\webpack:\\~\\babel-polyfill\\~\\core-js\\modules\\_microtask.js:18:1)\n    at process._tickCallback (internal/process/next_tick.js:61:11)","time":{"start":1665455027013,"stop":1665455029056,"duration":2043}},{"uid":"7340251bc6e3332f","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to remote server. Original error: Error: socket hang up\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to remote server. Original error: Error: socket hang up\n    at JWProxy.command$ (C:\\Program Files (x86)\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\jsonwp-proxy\\proxy.js:238:13)\n    at tryCatch (C:\\Program Files (x86)\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\babel-runtime\\regenerator\\runtime.js:67:40)\n    at GeneratorFunctionPrototype.invoke [as _invoke] (C:\\Program Files (x86)\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\babel-runtime\\regenerator\\runtime.js:315:22)\n    at GeneratorFunctionPrototype.prototype.(anonymous function) [as throw] (C:\\Program Files (x86)\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\babel-runtime\\regenerator\\runtime.js:100:21)\n    at GeneratorFunctionPrototype.invoke (C:\\Program Files (x86)\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\babel-runtime\\regenerator\\runtime.js:136:37)\n    at run (C:\\Program Files (x86)\\Appium\\resources\\app\\webpack:\\~\\babel-polyfill\\~\\core-js\\modules\\es6.promise.js:75:1)\n    at C:\\Program Files (x86)\\Appium\\resources\\app\\webpack:\\~\\babel-polyfill\\~\\core-js\\modules\\es6.promise.js:92:1\n    at flush (C:\\Program Files (x86)\\Appium\\resources\\app\\webpack:\\~\\babel-polyfill\\~\\core-js\\modules\\_microtask.js:18:1)\n    at process._tickCallback (internal/process/next_tick.js:61:11)","time":{"start":1665454932528,"stop":1665454934542,"duration":2014}},{"uid":"df259ba1e6fdbe90","status":"passed","time":{"start":1665453201844,"stop":1665453204053,"duration":2209}}],"categories":[],"tags":[]},"source":"8f979bf0657593ab.json","parameterValues":[]}